cfy_manager/components/cli/cli.py

```diff
@@ -47,6 +51,9 @@ def _set_colors(self, is_root):
         common.run([cmd], shell=True)
 
     def start(self):
+        self._start(installing=True)
```

Comment:
Why not just add the installing parameter to start, defaulted to True?

Reply:
Correct, changed